### DISCUSSION TITLE
TorchRL support

### DISCUSSION THREAD

**Discussion by vmoens** [ai4co/rl4co#127] [General]
Hi guys!
Great work on the library.
Just leaving this here to let you know that we're happy to support the effort in any way we can: if you spot issues or missing features in torchrl / tensordict just let us know.
If you'd like to be integrated in some design decisions feel free to let us know too! We'll ping you when relevant PRs are out.

**Comment by fedebotu** [comment]
Hi @vmoens! Thanks for dropping by. Of course we would love to collaborate with you 🚀
We have some features in mind that could be helpful for us, we will soon organize some ideas and share them so we can discuss 😁

**Comment by fedebotu** [comment]
(Just moved this issue to discussions)
As discussed in our previous meeting with @DavideTr8 , let's keep in touch! Also in the near future, we may want to officially integrate into the TorchRL API with a separate library of native TorchRL for environments :smile:
